code/parse/sexp.cpp

```diff
@@ -20920,18 +20735,23 @@ void sexp_set_training_context_fly_path(int node)
 	if (wp_list == NULL)
 		return;
 
+	int distance = eval_num(CDR(node));
+
 	Training_context |= TRAINING_CONTEXT_FLY_PATH;
 	Training_context_path = wp_list;
-	Training_context_distance = (float) atof(CTEXT(CDR(node)));
+	Training_context_distance = (float)distance;
 	Training_context_goal_waypoint = 0;
 	Training_context_at_waypoint = -1;
 }
 
 void sexp_set_training_context_speed(int node)
 {
+	int min = eval_num(node);
+	int max = eval_num(CDR(node));
+
 	Training_context |= TRAINING_CONTEXT_SPEED;
-	Training_context_speed_min = eval_num(node);
-	Training_context_speed_max = eval_num(CDR(node));
+	Training_context_speed_min = min;
+	Training_context_speed_max = max;
```

Comment:
Is this change really useful?
The additional variables min and max, don't really add any information to people reading the code.

Reply:
The upcoming PR will use them.  This change makes that PR cleaner.